reference-architecture/osp-cli/ch5.8.5_configure_docker_storage_ansible.sh

```diff
@@ -3,6 +3,6 @@ cat <<EOF >docker-storage-setup
 DEVS=/dev/vdb
 VG=docker-vg
 EOF
-ansible nodes -i inventory -m copy -a "src=docker-storage-setup dest=/etc/sysconfig/docker-storage-setup force=yes"
+ansible app-nodes,infra-nodes -i inventory -m copy -a "src=docker-storage-setup dest=/etc/sysconfig/docker-storage-setup force=yes"
```

Comment:
As masters are also nodes but they don't have a dedicated disk for docker images, the docker-storage-setup file is only copied to the hosts they have the dedicated disk.